examples/resources/juju_kubernetes_cloud/import.sh

```diff
@@ -0,0 +1,2 @@
+# Kubernetes clouds can be imported by using kubernetes cloud name.
+$ terraform import  juju_kubernetes_cloud.kubernetes-cloud-name
```

Comment:
todo: add the example to read the cloud

to import the cloud, I had to:
`terraform import juju_kubernetes_cloud.test-k8s-cloud test-k8s-cloud:test-k8s-cloud`